packages/wrangler/src/pages/publish.tsx

```diff
@@ -95,6 +96,18 @@ export const Handler = async ({
 		throw new FatalError("Must specify a directory.", 1);
 	}
 
+	const { defaulted } = yargs.parsed as unknown as {
+		defaulted: Record<string, boolean>;
+	};
+	const bundleDefaulted = defaulted["bundle"];
+	const noBundleDefaulted = defaulted["no-bundle"];
```

Comment:
This is a horrible hack to check if we are using the default values 😓 , @GregBrimble can you think of something better?